pkg/controller/associated/android.go

```diff
@@ -39,7 +39,8 @@ func (c *Controller) getAndroidData(region string) ([]AndroidData, error) {
 		return nil, fmt.Errorf("unable to lookup realm: %w", err)
 	}
 
-	apps, err := c.db.ListActiveAppsByOS(realm.ID, database.OSTypeAndroid)
+	scopes := []database.Scope{database.WithAppOS(database.OSTypeAndroid)}
```

Comment:
Nit, but since this is the only scope, why not just:

```go
apps, err := c.db.ListActiveApps(realm.ID, database.WithAppOS(database.OSTypeAndroid))
```

throughout?

Reply:
sure we can do that